### ISSUE TITLE
Calendar panel

### ISSUE THREAD

**Issue by SamKr** [heyajohnny/afvalinfo#141]
Heya Johnny, first off; thanks for the great integration! Works like a charm 👍 

I was wondering if there are any plans to integrate the events with the calendar panel? Or would you know a way to easily do that ourselves?

Thanks

**Comment by heyajohnny** [comment]
Hi SamKr,

I don't think it's easy to do... 

The way I build this integration was mainly to have up-to-date pickup dates for your trash. 
On a regularly base, the integration checks the webpage/API of the local waste collectors to only retrieve the first upcoming date of any given trash type.

The way these calendars work, is that you can import a batch of activities, preferably only once a year, in this case.
So you need to get the pickup dates for the whole year and import it in the calendar. The down side to this is that you don't have up-to-date pickup dates. If you did an import in January a lot could've happened in the mean time (let's say COVID or a change of the pickup route) that could've mixed up the pickup dates. In this case, your pickup dates might be wrong...

I know that some of the waste collectors offer an option to export/import the dates into a calendar, so that might be an option for you.

**Comment by SamKr** [comment]
Alright, makes sense. Still sticking with yours since it works really well, and the calendar panel was just a nice-to-have. 

Thanks for the extensive answer :)
